equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/mapper/CrmMapper.java

```diff
@@ -43,6 +44,13 @@ default String convertDecisionReason(CrmDecisionReasonDetailsModelInterface mode
                 model.getFull_grant_notes() : model.getReason_details();
     }
 
+    @Nullable
+    default Boolean convertIntegerToBoolean(Integer i) {
+        if (i == null) return null;
```

Comment:
can we change to one liner as below?
`return i==null ? null : (i > 0);`

Reply:
I don't like 1 liners returning null or false as 1st option. It mislead to think that null is the most common when it is not